app/models/audit/event/authn.rb

```diff
@@ -81,8 +84,14 @@ def attempted_action
       def auth_stuctured_data
         { authenticator: @authenticator_name }.tap do |sd|
           sd[:service] = service_id if @service
+          sd[:user] = username
         end
       end
+
+      # In case of success returns the role performing the action, otherwise conceals it for security concerns
```

Comment:
i think that this comment is redundant. the code in line 93 is self-explanatory

Reply:
I added this comment because I was asked to by another reviewer. I tend to agree that comment is not necessary for such a simple function